lib/routes/content.js

```diff
@@ -53,29 +48,20 @@ const routes = application => {
      * @param {object} next - Express callback
     */
     app.get(`/${config.content.base}/:type`, (req, res, next) => {
-      const type = utils.singleItem('id', req.params.type.toLowerCase(), types);
+      const good = utils.content.check.url(req, types, allFlows);
```

Comment:
Seeing this here, I would really love to see this as a promise, especially given that `routes` is a promise; all of the errors could roll up nicely